tools_iuc.yaml

```diff
@@ -1599,6 +1599,14 @@ tools:
     owner: iuc
     tool_panel_section_label: Mapping
 
+  - name: coverm_contig
+    owner: iuc
+    tool_panel_section_label: Mapping
```

Comment:
I am not it is the best section for CoverM. I would have put it in "Metagenomic Analysis"
In any case, thanks a lot for adding it. I will tell Magnus and Valerie that the tool is there

Reply:
You are right, sorry; I didn't check the CoverM README.

Reply:
Can you change the category here ... I will fix it on the server.

Reply:
Sure.